piecrust-uplink/src/types.rs

```diff
@@ -83,6 +94,8 @@ impl core::fmt::Debug for ModuleId {
     }
 }
 
+/// A RawQuery is a host query that doesn't care about types and only operates
+/// on raw data.
```

Comment:
A `RawQuery` is not a "host query". It's a normal query (contract call) that doesn't care about types.